src/main/java/hlf/java/rest/client/config/FabricProperties.java

```diff
@@ -1,13 +1,14 @@
 package hlf.java.rest.client.config;
```

Comment:
Looks like the formatter check is not working as expected. These files have no change in them but still shows up.

Reply:
Noted, will fix.

Reply:
This is addressed.